exceptions.py

```diff
@@ -0,0 +1,17 @@
+import game
+
+
+class GameOver(Exception):
+    """Game saving mechanics"""
+
+
+    def __init__(self):
+        self.save_result()
+
+    def save_result(self):
+        with open('scores.txt', 'a') as file:
+            return file.write(f"{game.player.name} - {game.player.score} \n")
```

Comment:
do not import game here.
You can pass player name and score as parameters in __init__ method of exception